runtime/src/runtime/hash_algorithm.rs

```diff
@@ -34,7 +34,16 @@ impl HashAlgorithm for FvmHashSha256 {
         let mut rval_digest: HashedKey = Default::default();
         let mut hasher = RuntimeHasherWrapper::default();
         key.hash(&mut hasher);
-        fvm::crypto::hash_into(SupportedHashes::Sha2_256, &hasher.0, &mut rval_digest);
-        rval_digest
+
+		// TODO :: Enable this when moving to
```

Comment:
Ah, sorry, I hadn't realized we hadn't ported that. yet.

Reply:
😄 , no problem. That was causing the CI Build error 👍🏾